src/stronghold.hs

```diff
@@ -5,40 +5,38 @@ module Main where
   This file should only define Stronghold's API.
 -}
 
-import Prelude hiding (mapM)
-
-import Data.Monoid (mempty, mconcat, Endo(Endo), appEndo)
-import Data.Maybe (fromJust, listToMaybe)
-import Data.ByteString (ByteString)
-import qualified Data.ByteString.Char8 as BC
-import qualified Data.ByteString.Base16 as Base16
-import Data.Text (Text, unpack)
-import qualified Data.Text as Text
-import Data.Text.Encoding (decodeUtf8)
-import qualified Data.Aeson as Aeson
-import qualified Data.HashMap.Strict as HashMap
-import Data.Time.Clock (getCurrentTime, UTCTime)
-import Data.Traversable (mapM)
-
 import Control.Applicative ((<$>), (<*>), (<|>), empty)
+import Control.Concurrent (myThreadId, throwTo)
+import Control.Exception (SomeException, try, tryJust)
 import Control.Monad (foldM, join)
 import Control.Monad.IO.Class (liftIO)
-import Control.Exception (tryJust, try, SomeException)
-import Control.Concurrent (throwTo, myThreadId)
-
-import Snap.Core
-import Snap.Http.Server
-
 import Crypto.Hash.SHA1 (hash)
-
+import Data.ByteString (ByteString)
+import Data.Maybe (fromJust, listToMaybe)
+import Data.Monoid (Endo(Endo), appEndo, mempty, mconcat)
+import Data.Text (Text, unpack)
+import Data.Text.Encoding (decodeUtf8)
+import Data.Time.Clock (UTCTime, getCurrentTime)
+import Data.Traversable (mapM)
+import Prelude hiding (mapM)
+import Snap.Core (Method(GET, POST), Snap)
+import StoredData (JSON)
 import System.Environment (getArgs)
-import System.IO (Handle, stdout, stderr)
+import System.IO (Handle, stderr, stdout)
+import Util (Path)
 
-import qualified ZkInterface as Zk
+import qualified Data.Aeson as Aeson
+import qualified Data.ByteString.Base16 as Base16
```

Comment:
I think B16 might be more consistent with other ByteString abbreviations, e.g. B, BL, BC. I would argue that `Base16` doesn't make it as clear that it's a ByteString.